projects/storefrontlib/src/lib/cms-lib/breadcrumb/breadcrumb.component.html

```diff
@@ -1 +1,6 @@
-<div class="breadcrumb">(breadcrumb)</div>
+<nav>
+  <a *ngFor="let crumb of (crumbs$ | async)" [routerLink]="crumb.link">
```

Comment:
Just a note, that this breadcrumb will have to take into account the configurable router links. But the implementation highly depends on our further strategy for configurable routing.

Reply:
Totally. This PR was mainly to introduce the page title. When we work on the crumbs, we should take this into account for sure.